irods_python_ci_utilities/irods_python_ci_utilities.py

```diff
@@ -89,14 +90,16 @@ def install_os_packages_from_files_yum(files):
     subprocess_get_output(args, check_rc=True)
 
 def install_os_packages_from_files_zypper(files):
-    install_os_packages_zypper(files)
+    args = ['sudo', 'zypper', '--no-gpg-checks', '--non-interactive', 'install'] + list(files)
```

Comment:
do we have a good reason to disable the encryption?

Reply:
if the files are in a file system like a volume mount then there are no keys associated with them.

Reply:
ah, so zypper can just install files from a local directory.